src/input/datetime.rs

```diff
@@ -496,6 +496,11 @@ impl TzInfo {
         None
     }
 
+    fn fromutc<'p>(&self, py: Python<'p>, dt: &'p PyDateTime) -> PyResult<&'p PyDateTime> {
+        dt.call_method1("__add__", (self.utcoffset(py, py.None().as_ref(py))?,))?
```

Comment:
@davidhewitt would be nice having support for `PyDateTime + PyDelta -> PyDateTime`

Reply:
So the problem with Rust operators is that they assume they can't fail. I think we can't implement this operator because we can't guarantee `__add__` won't fail for `PyDateTime` subclasses.

Reply:
Thanks! Well, it is what it is then 🤷🏻‍♂️

Reply:
We could have `dt.add(x)` though, if that's better? 😂